html/Corporate/js/video/video.js

```diff
@@ -0,0 +1,14 @@
+const videoBtn = document.querySelector('.banner__video-btn');
+const video = document.querySelector('.banner__video');
```

Comment:
any access to your DOM should be wrapped to DOMContentLoaded callback. 
Not each one, but the whole script that's going to access DOM